src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Completing order method was called with userId={}", userId);
```

Comment:
as I mentioned above - you should try to keep your logs as short as possible
```suggestion
        logger.debug("Completing order for userId={}", userId);
```